.ci/build-kit/install_and_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+cmake \
+    -B build \
+    -S "$EXT_MOUNT/source" \
+    -G Ninja \
+    -DBUILD_TESTING_EVSE_SECURITY=ON \
```

Comment:
```suggestion
    -DBUILD_TESTING=ON \
```

I thought running a workflow added by a fork is not possible. I don't understand why it is now.

This change should probably fix
https://github.com/EVerest/libevse-security/actions/runs/7817099985/job/21329584371?pr=25

Locally the unit tests pass..